### DISCUSSION TITLE
DNS issues - In local network

### DISCUSSION THREAD

**Discussion by Sonnex7** [dani-garcia/vaultwarden#4139] [Q&A] [answered]
My setup looks like this:

- nginx proxy manager
- vaultwarden 
[docker-compose.yml.txt](https://github.com/dani-garcia/vaultwarden/files/13588544/docker-compose.yml.txt)
- duckdns account
- dns challenge with duckdns
- pihole for DNS

I have the problem that my ****.duckdns.org cannot be resolved.
I get the following error:
DNS_PROBE_FINISHED_NXDOMAIN

When i looked through the documentation I found this:
https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs

I think i have to configure an exception at my pihole but I am not sure.
I tried to add a Local DNS Record but it did not change anything. 

Can someone help?

**Comment by ghost** [comment]
Did you try a public resovler? If you're not familiar with `dig` or `nslookup` command, try online service:
nslookup.io

**Reply by Sonnex7** [reply]
I tried both and the online service none of them can resolve it.

**Dig**
dig *****.duckdns.org

; <<>> DiG 9.18.12-0ubuntu0.22.04.3-Ubuntu <<>> ****.duckdns.org
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 29106
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;****.duckdns.org.		IN	A

;; Query time: 3 msec
;; SERVER: 192.168.178.1#53(192.168.178.1) (UDP)
;; WHEN: Wed Dec 06 16:46:31 UTC 2023
;; MSG SIZE  rcvd: 36

**nslookup**
nslookup ****.duckdns.org
Server:		192.168.178.1
Address:	192.168.178.1#53

Non-authoritative answer:
*** Can't find *****.duckdns.org: No answer

But should not this behaviour be normal?

**Comment by Sonnex7** [comment, marked as answer]
ok finally solved the problem:

Had to configure an exception at my fritzbox.
DNS-Rebind-Protection was the problem.
![2023-12-06 18_21_43-FRITZ!Box 6660 Cable](https://github.com/dani-garcia/vaultwarden/assets/153077371/9d70279a-fdee-40fb-88e8-f1fd4da50cc5)
